tcms/bugs/views.py

```diff
@@ -227,6 +228,19 @@ def post(self, request):
                 add_comment([bug], _("*bug reopened*"), request.user)
             bug.save()
 
+            from tcms.core.utils.mailto import mailto
+
+            mailto(
+                template_name="email/post_bug_save/email_update.txt",
+                recipients=[bug.assignee.email, bug.reporter.email],
+                subject=_("UPDATE: Bug #%(pk)d - %(summary)s")
+                % {"pk": bug.pk, "summary": bug.summary},
+                context={"bug": bug,
+                         "comment": form.cleaned_data["text"],
+                         "updated_by": request.user.username,
+                         "updated_at": now()}
+            )
+
```

Comment:
All emails are handled in signal handlers which can be turned on/off and even extended in future, e.g. email + Slack notification. 

The POC is good but you have to turn this into a signal handler (custom signal, something like bugs_comment_added) and trigger the signal after a comment has been added.